### PR TITLE
release: v2.0.0

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,7 +106,7 @@
     ]
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=16.0.0"
```
Comment (alejandromumo) [inline package.json:109]: Why is this needed?

Comment (kpsherva) [reply package.json:109]: because otherwise we get a message about unsupported engine. Until we fully move to webpack 5, we need also to support node16
